app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java

```diff
@@ -308,7 +310,8 @@ private void editFailed(@NonNull Throwable caught, boolean logError) {
 
         @Override
         public void onHelpClick() {
-            startActivity(DescriptionEditHelpActivity.newIntent(requireContext()));
+            UriUtil.visitInExternalBrowser(getContext(),
```

Comment:
Hint: `FeedbackUtil.java` contains numerous convenience functions for visiting specific external pages, e.g. the FAQ page, the privacy policy, etc.  We can simply add another function there that goes to this page.

Reply:
Right! That will be cleaner. Will do this!